src/main/java/com/auto/ht/helpers/iFrameHelper.java

```diff
@@ -0,0 +1,181 @@
+package com.auto.ht.helpers;
+
+import com.codeborne.selenide.CollectionCondition;
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.By;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+
+public class iFrameHelper {
+    private static final Logger log = LoggerFactory.getLogger(iFrameHelper.class);
+
+    // Common iframe selectors that might intercept clicks
+    private static final String[] COMMON_FRAME_SELECTORS = {
+            "iframe#preview-notification-frame",
+            "iframe.st_preview_frame_banner",
+            "iframe[title='smtiframetitle44']",
+            "iframe.notification-frame",
+            "iframe.ad-frame",
+            "iframe[style*='position:fixed']"
+    };
+
+    // Common close button selectors
+    private static final String[] CLOSE_BUTTON_SELECTORS = {
+            "button.close-notification",
+            ".close-button",
+            ".close-icon",
+            ".btn-close",
+            "[aria-label='Close']",
+            "[title='Close']",
+            "button.st_close",
+            ".st-close",
+            ".x-button"
+    };
+
+    /**
+     * Checks and handles any intercepting frames on the page
+     */
+    public static void handleInterceptingFrames() {
```

Comment:
we dont handle iframe like this. You **must** know which iframe you're suppose to switch!